drivers/platform/maxim/max32660/maxim_spi.c

```diff
@@ -151,6 +157,9 @@ static int _max_spi_config(struct no_os_spi_desc *desc)
 		goto err_init;
 	}
 
+	spi_pins.vssel = eparam->vssel;
```

Comment:
Just a remark, for the sake of completeness, if vssel is not initialized by driver API user, its value will default to 0 which means VDDIO (not VDDIOH). I think this is good behavior.